Ticket

```diff
@@ -31,6 +33,20 @@ public class Ticket extends TicketAbs {
     		return price_ar[movie][cinema][client.age][showtime];
     	else return -1;
     }
+    
+    public void create_ticket_price(double[][][][]price_ar) {
```

Comment:
Just a slight note on the naming convention, in Java we'll name methods createTicketPrice instead of using underscores. Functionally they're the same, but it just looks better on the Javadoc